plots/laws/thermodynamics/fuel_consumption_and_engine_power.py

```diff
@@ -0,0 +1,82 @@
+from sympy import symbols, Eq
+from sympy.plotting import plot
+from sympy.plotting.plot import MatplotlibBackend
+from symplyphysics import print_expression
+from examples.thermodynamics import \
+    energy_from_combustion_of_gasoline as combustion_energy_example
+
+# Note: fuel_consumption = volume_of_fuel / distance
+fuel_consumption = symbols("fuel_consumption")
+
+# Create efficiency factor values from 15% to 75% in increments of 15%
+EFFICIENCY_FACTOR_1 = 0.15
+EFFICIENCY_FACTOR_2 = 0.3
+EFFICIENCY_FACTOR_3 = 0.45
+EFFICIENCY_FACTOR_4 = 0.6
+EFFICIENCY_FACTOR_5 = 0.75
```

Comment:
Why did you abandon your previous implementation, with automatic generation of the factors?

Reply:
I thought about this: 

1) Since the reuse of the code (for constructing each of the plots using a loop) has been removed, the list according to which the cycle was conducted earlier is not needed. 

2) Since each plot is independent, in this format it is easier to change the efficiency factor for each of them at your discretion. That is, the convenience of setting up plots.

Reply:
I told to not reference other examples. You can use loops in your example.

Reply:
If you want to setup efficiency factors independently, you can do it like this:
`effective_factors = [0.15, 0.3, 0.45]`

Reply:
> I told to not reference other examples. You can use loops in your example.

Sorry, I misunderstood the remark. I will keep this in mind.